packages/sdk/src/dataProtector/sharing/subgraph/getProtectedDataById.ts

```diff
@@ -46,6 +50,10 @@ export async function getProtectedDataById({
       isForSale: boolean;
       rentals: Array<{ id: string }>;
     };
+    rentalParam: {
+      price: number;
+      duration: number;
+    };
```

Comment:
I'd say it should be a different query.
It's only destructured once (in `rentProtectedData.ts`) so it means that for all other callers we would get these rentalParam for nothing.

wdyt?

Reply:
may be you can develop the tree last functions that we have to do and see after if it's interesting move this peace of code ?

Reply:
Yes, agreed.